tests/conftest.py

```
@@ -0,0 +1,101 @@
+    CONF_VALUE,
+    CONF_VALUE_TYPE,
+    CONF_VARIABLE_ID,
+    CONF_YAML_VARIABLE,
+    DOMAIN,
+)
+from tests.types import ConfigEntryFactory
+
+
+@pytest.fixture(autouse=True)
+def auto_enable_custom_integrations(
+    enable_custom_integrations: None,
```

Ruff (0.16.0)

[warning] Unused function argument: `enable_custom_integrations`

(ARG001)